lib/engine/src/trap/frame_info.rs

```diff
@@ -42,6 +42,15 @@ pub struct GlobalFrameInfo {
     ranges: BTreeMap<usize, ModuleInfoFrameInfo>,
 }
 
+/// Returns whether the `pc`, according to globally registered information,
+/// is a wasm trap or not.
+pub fn is_wasm_pc(pc: usize) -> bool {
+    let frame_info = FRAME_INFO.read().unwrap();
+    let module_info = frame_info.module_info(pc);
+    let is_wasm_pc = module_info.is_some();
+    is_wasm_pc
```

Comment:
It's very likely that Clippy will suggest:

```suggestion
    module_info.is_some()
```